enterprise/models.py

```diff
@@ -118,6 +118,13 @@ class Meta:
         )
     )
 
+    contact_email = models.EmailField(
```

Comment:
hmm...in general I don't really like the idea of adding fields like this directly to the "aggregate root" of a domain...it seems to me we'd be better off adding an EnterpriseCustomerContactInfo model which could contain this email field and an FK back to the EnterpriseCustomer...